### ISSUE TITLE
Skip to content anchor / h1 behaviour on articles

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#6243]
**Describe the bug**
For some articles, there are items prior to the h1, e.g. this one: https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo
In these cases, if navigating directly to the h1 on a screen reader or using the skip to content link with a keyboard, this will bypass these items, thus missing article content before the h1.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo2. use skip to content link
3. see it go straight to the h1, skipping past the image that is prior to it. 

**Expected behaviour**

- If h1/headline block is first - do nothing (and skip to content link will work as it currently is)
- If h1/headline block is not first - add visually hidden h1 at the top, transform h1 added in optimo to strong HTML tag (skip to content to go to visually hidden h1) we currently do this on MAP pages see: https://www.bbc.com/mundo/media-54661776
- If no h1/headline block - add visually hidden h1 at the top using the page title removing the service brand - is this the SEO headline? Skip to content goes here

**Scenario:** 
Given the first block in article is a h1/headline 
When I skip to content 
then the h1 is the first element on the page and I get taken here 
e.g. https://www.test.bbc.com/pidgin/articles/cwkvd1410e9o

Given the h1/headline block is not the first element in the article
when I skip to content 
then I go to a visually hidden h1 before any other content in the main landmark
and the visually hidden h1 uses the headline text 
and the visual headline is a strong html tag with the headline text
e.g. https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo

Given no h1 headline is added to an Optimo article 
when I skip to content
then I go to a visually hidden h1 before any other content in the main landmark
and the h1 uses the SEO headline from Optimo 
e.g. https://www.test.bbc.com/pidgin/articles/c5l7vpz6z35o

**Screenshots**
N/A

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
Prior to picking up this ticket, we should discuss the approach. 
We should document the decision made, so we can ensure other page types (other than Optimo Articles) take the same approach.

**Comment by mukama** [comment]
Hi @sareh. The link you provided seems to be broken - https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo2

**Comment by LilyL0u** [comment]
> Hi @sareh. The link you provided seems to be broken - https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo2

You might have found this now anyway, but the URL works if you remove the 2 from the end https://www.test.bbc.com/pidgin/articles/cq16zlmz9ddo

**Comment by LilyL0u** [comment]
Is the plan here to make the skip to content anchor first go to the H1 and then back up to the content above it? Like on MAPs

**Comment by mukama** [comment]
We tested a 2 approaches and documented our finding here: https://paper.dropbox.com/doc/Skip-to-content-anchor-always-go-to-the-h1-even-when-there-is-content-prior-to-it--BTzbJlZNcFRtqjrNdvEEGCquAg-yZvJamtp1ARDSyWq3DKPt

**Comment by aeroplanejane** [comment]
@greenc05 @mukama Could the workaround here could be the use of a hidden h1 like on MAPs? 
https://www.bbc.com/pidgin/tori-58761997
So we always have the h1 at the top of the page, despite the placement of the headline. I believe this is also the approach adopted by the webcore team.

**Comment by mukama** [comment]
Yes @aeroplanejane. Let's do that but only for articles that do not start with h1

**Comment by aeroplanejane** [comment]
@mukama @greenc05 I've updated the issue with the agreed solution and acceptance criteria above. Please shout if anything is missing or you have any further questions.
